subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -98,40 +118,51 @@ static int get_published_payload(struct mqtt_client *client, u8_t *write_buf,
  * @param[in] fota_status next_state  The next state the device's fota operation
  *				      is going to enter. Reported to the status
  *				      details of the Job Execution.
- * @param[in] version_number  The expected version number of the Job Execution.
+ * @param[in] offset The number of bytes which have been downloaded of the
```

Comment:
I suggest renaming to something like `download_progress`. The docs can be

"How many bytes of the upgrade have been downloaded. Will be reported to [other side] through the status details field of the Job Execution."

Fill in proper word for the [other side] ("server" or whatever is correct).

Reply:
maybe make it as simple as progress ?